app/_hub/kong-inc/proxy-cache-advanced/_index.md

```diff
@@ -86,6 +86,13 @@ params:
       datatype: boolean
       description: |
         When enabled, respect the Cache-Control behaviors defined in [RFC7234](https://tools.ietf.org/html/rfc7234#section-5.2).
+    - name: ignore_uri_case
+      required: false
+      default: false
+      value_in_examples: null
+      datatype: boolean
+      description: |
+        When enabled, the incoming request will be lowercase before feeding into the Cache-Key calculation.
```

Comment:
Please revert this, proxy-cache-advanced is a different plugin and will be treated separately

Reply:
Ah alright, didn't know there was some other process for that. I assumed that this was supposed to be added there as well :-)

Reply:
resolved with d03e05e5e33933a92c49c069f96971934ededc78